README.md

```diff
@@ -92,43 +94,321 @@ It even works on MacOSX in addition to the OS scroll acceleration,
+You can do both of these steps at once in one command by running:
+```bash
+scroll-accelerator --restart-daemon --multiplier 0.8
+```
+Note that the new config is automatically saved when using `--restart-daemon`
+or `--install-daemon`
+and therefore the `--save-config` option is not needed.
+
+You can also modify the configuration by editing the file
+`~/.config/scroll-accelerator/config.yaml` directly.
+
+### Stopping/uninstalling the daemon
+
+To stop the daemon, you can use the `--stop-daemon` option.
+```bash
+scroll-accelerator --stop-daemon
+```
+This will stop the daemon temporarily, but it will still be restarted on
+boot.
+
+To uninstall the daemon, you can use the `--uninstall-daemon` option.
+```bash
+scroll-accelerator --uninstall-daemon
+```
+This will stop the daemon and uninstall it. Note that this will not remove
+the configuration file.
+
+### Getting the status/log of the daemon
+
+To get the status of the daemon, you can use the `--daemon-status` option.
+```bash
+scroll-accelerator --daemon-status
+```
+This will print the status of the systemd unit associated with the daemon.
+
+To get the log of the daemon, you can use the `--daemon-log` option.
+```bash
+scroll-accelerator --daemon-log
+```
+This will print the log of the daemon.
+
+Note that this will be fairly empty by default.
+To get more verbose output, you can use the `--daemon-verbosity` option
+when installing or restarting the daemon:
+```bash
+scroll-accelerator --restart-daemon --daemon-verbosity 1
+```
+Then, after scrolling a few times, you can use `--daemon-log` again to see
+the log.
+It should now show a more verbose log, including the scroll events and the
+acceleration.
+
+## Acceleration parameters
+
+The following acceleration parameters determine the behavior of the scroll
+accelerator
+and can be set using the command line options or the configuration file.
+
+* `exp`: the exponential factor. This controls how fast the scroll speed
+  increases.
+* `multiplier`: the scalar multiplier. This is multiplied by the scroll
+  speed.
+* `threshold`: the threshold. This controls the minimum scroll speed.
+
+The exact formula for computing the target scroll speed is:
+```python
+m = (user_scroll_speed - threshold) ** exp
+target_scroll_speed = user_scroll_speed + m * multiplier
+```
 
-If you found values that work for you, you can install
-the script as a systemd user unit (only on Linux):
+## Uninstallation
 
-    ./install-daemon.py
+To uninstall `scroll-accelerator`, first uninstall the daemon:
+```bash
+scroll-accelerator --uninstall-daemon
+```
+
+Then you can uninstall the application using whichever package manager was
+used to install it.
+For pipx:
+```bash
+pipx uninstall scroll-accelerator
+```
+
+For `pip`:
+```bash
+pip uninstall scroll-accelerator
+```
+
+Note that this will not remove the configuration file.
+To remove the configuration file, you can run:
+```bash
+rm -rf ~/.config/scroll-accelerator
+```
+
+## Development/Contributing
```

Comment:
I think this whole section should go into a separate document. (DEVELOPMENT.md or so?)